src/phoenix/server/api/input_types/DimensionFilter.py

```diff
@@ -54,20 +55,12 @@ class DimensionFilter:
     shapes: Optional[List[DimensionShape]] = UNSET
 
     def __post_init__(self) -> None:
-        setattr(self, "types", _ensure_list(self.types))
-        setattr(self, "shapes", _ensure_list(self.shapes))
+        self.types = ensure_list(self.types)
+        self.shapes = ensure_list(self.shapes)
```

Comment:
oh I see now

Reply:
Maybe you could be a tad safer in ensure_list that it only falls back to a list if the value is `None` or `Unset` so that you don't fall back for everything

Reply:
Yeah I get your intention - just can't guarantee it will be used by someone correctly. Just was thinking you could narrow the type signature of the parameter to ensure proper usage.

Reply:
got it. yea, the type signature has been updated